server/routes/home.js

```diff
@@ -8,6 +9,25 @@ module.exports = function Index({ authenticationMiddleware, userService, offende
 
   router.get(
     '/',
```

Comment:
I did a redirect here to allow for users who may have both roles and want to see either list page at some point in the near future!